tests/schedule.go

```diff
@@ -130,13 +131,17 @@ func (s *ScheduleFunctionalSuite) TestBasics() {
 	wt2 := "sched-test-basics-wt2"
 
 	// switch this to test with search attribute mapper:
-	// csa := "AliasForCustomKeywordField"
```

Comment:
btw, I left this here a while ago.. we should probably do this for real now, like run the test twice, once with "real" search attributes and once with mapped ones. can be in a followup PR

Reply:
Actually, the current test setup is already using aliases.